dadapy/metric_comparisons.py

```diff
@@ -611,3 +611,64 @@ def return_label_overlap_selected_coords(self, labels, coord_list, k=30):
             overlaps.append(overlap)
 
         return overlaps
+
+    def return_inf_imb_causality(
+        self, cause_present, effect_present, effect_future, weights=[1], k=1
```

Comment:
Error B006 can probably be solved simply writing weights=(1) instead of weights=[1]

Reply:
I directly removed the default value